packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -523,6 +523,66 @@ export class TestObjectProvider implements ITestObjectProvider {
 	}
 }
 
+export class TestObjectProviderWithVersionedLoad extends TestObjectProvider {
```

Comment:
Is there any reasoning why we extend `TestObjectProvider` instead of implementing `ITestObjectProvider`?

Reply:
I'm overriding the loadTestContainer method in TestObjectProvider and I think I need to use extends to do this.